Integrationtest of SSL Ciphers broken on Ubuntu

How to reproduce:
```
> ./gradlew integrationtest
...
PDSServerEncryptionTest > tls_1_3_must_be_accepted() FAILED
    java.net.ConnectException at PDSServerEncryptionTest.java:44

PDSServerEncryptionTest > sslv3_is_not_accepted() FAILED
    java.net.ConnectException at PDSServerEncryptionTest.java:59

PDSServerEncryptionTest > tls_1_2_must_be_accepted() FAILED
    java.net.ConnectException at PDSServerEncryptionTest.java:39

PDSServerEncryptionTest > tls_1_0_is_not_accepted() FAILED
    java.net.ConnectException at PDSServerEncryptionTest.java:54

PDSServerEncryptionTest > verified_ciphers_do_have_MAC_with_SHA_or_SHA1() FAILED
    java.io.IOException at PDSServerEncryptionTest.java:29
        Caused by: java.io.IOException at PDSServerEncryptionTest.java:29

PDSServerEncryptionTest > verified_ciphers_do_have_MAC_with_MD5() FAILED
    java.io.IOException at PDSServerEncryptionTest.java:34
        Caused by: java.io.IOException at PDSServerEncryptionTest.java:34

PDSServerEncryptionTest > tls_1_1_is_not_accepted() FAILED
    java.net.ConnectException at PDSServerEncryptionTest.java:49

ServerEncryptionTest > verified_ciphers_do_have_MAC_with_SHA_or_SHA1() FAILED
    java.io.IOException at ServerEncryptionTest.java:29
        Caused by: java.io.IOException at ServerEncryptionTest.java:29

ServerEncryptionTest > verified_ciphers_do_have_MAC_with_MD5() FAILED
    java.io.IOException at ServerEncryptionTest.java:34
        Caused by: java.io.IOException at ServerEncryptionTest.java:34
...
```

--- a/sechub-integrationtest/ciphertest.sh
+++ b/sechub-integrationtest/ciphertest.sh
@@ -1,4 +1,4 @@
-#!/usr/bin/bash
+#!/bin/bash
 
 # This script creates a json file containing openssl verification meta data
 # which is used inside java unit tests via SecurityTesthelper.java

